### DISCUSSION TITLE
api call for Passphrase Lockdown

### DISCUSSION THREAD

**Discussion by Tachaeon** [pglombardo/PasswordPusher#1679] [Q&A] [answered]
Would like to see this in a new version. RN there isn't a feature where i can stick a passphrase in a variable and send it through the api.

Thanks,
Tach

**Comment by pglombardo** [comment, marked as answer]
Hi @Tachaeon - which API client are you using?  The API supports the `password[passphrase]` option.  Here's the [API documentation](https://pwpush.com/api/1.0/passwords/create.en.html).

I should probably update this documentation to be a bit clearer.

Let me know which client you're using.

**Reply by Tachaeon** [reply]
thanks for the info I was reading this documentation https://github.com/pglombardo/PasswordPusher/wiki/Password-API

![image](https://github.com/pglombardo/PasswordPusher/assets/97981421/684cb075-8236-4184-89ea-126421f34be3)
